Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -481,6 +481,7 @@ - (nullable TSAttachment *)firstAttachmentIfAnyOfMessage:(TSMessage *)message
     if (message.attachmentIds.count == 0) {
         return nil;
     }
+    // TODO: Support multi-image messages.
```

Comment:
There's a bunch of these TODOs identifying possible follow-up work.  I tried to make them identical to facilitate pursuing them.